src/pykeen/training/lcwa.py

```diff
@@ -71,11 +70,25 @@ def __init__(
         # The type inference is so confusing between the function switching
         # and polymorphism introduced by slicability that these need to be ignored
         if self.target == 0:
-            self.score_method = self.model.score_h  # type: ignore
+            if self.model.use_inverse_triples:
+                self.score_method = self.model.score_h_inverse  # type: ignore
+                self.can_slice = self.model.can_slice_t
```

Comment:
do we need to check for can_slice anymore? I thought this was enabled everywhere now

Reply:
No, we don't need - that's likely an additional merge artifact 😅